recipes/alibabacloud-credentials/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "alibabacloud-credentials" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/alibabacloud_credentials-{{ version }}.tar.gz
+  sha256: 65f0e1d5fed1b0751dc56b9dc565144e528803185193e4771a21ef9d9d138d59
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - alibabacloud-tea
+
+test:
+  imports:
+    - alibabacloud_credentials
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/aliyun/credentials-python
+  summary: The alibabacloud credentials module of alibabaCloud Python SDK.
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
The license file I have included is what is linked to (rather than explicitly included) in the README.md files in each repo.